.github/dependabot.yml

```
@@ -8,3 +8,8 @@ updates:
   ignore:
   - dependency-name: docs.validation
   - dependency-name: docs.validation.analyzer
+  - package-ecosystem: "github-actions"
```

Thanks! Should we remove the indention here? otherwise this array is attached to the `ignore` object. 
```suggestion
- package-ecosystem: "github-actions"
```